thirdeye/thirdeye-frontend/app/pods/components/filter-bar/component.js

```diff
@@ -165,13 +162,13 @@ export default Ember.Component.extend({
   actions: {
 
     /**
+     * Filters results in the events table depending on a selected event type in the filter block
      * Expands/collapses a filter block
      * @method filterByEvent
      * @param {Object} clickedBlock - selected filter block object
      */
     selectEventType(clickedBlock) {
-      const { entities, onSelect } = this.getProperties('entities', 'onSelect');
-      const cachedHeader = this.urnsCache[clickedBlock.header];
+      const filterUrns = this.get('filterUrns');
       let filterBlocks = this.get('config');
```

Comment:
Any reason we're using `let`here? I don't see the variable being modified latter on.
Also this can be reduced to a one line: 
```
const {
filterUrns,
config: filterBlocks
} = this.getProperties('filterUrns', 'config')
```

Reply:
see `Assigning to new variable names` in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment